strawberryfields/gbs/plot.py

```diff
@@ -345,49 +346,72 @@ def points(
     return f
 
 
-LIGHT_BLUE = "#94cecf"
-
-BAR_COLOUR = LIGHT_BLUE
-
-
-def spectrum(energies: list, bins: Optional[int] = 0):  # pragma: no cover
-    """Plots a spectrum based on input sampled energies.
+def spectrum(
+    energies: list, gamma: float = 100.0, xmin: float = None, xmax: float = None
```

Comment:
No worries if this was discussed/decided, but I would have the addition of the Lorentzian curve being optional. Still `gamma` can default to a width of `100`, but if the user inputs something like `None`, they would be able to get just the histogram without the curve.

Reply:
This is a reasonable suggestion although I should say that plotting just the bar chart (histogram) form energies provided by ``vibronic.energies`` requires 2 line of code. If the user just wants the bar charts, it can be done with minimum work instead of using our ``plot.spectrum`` function.

Reply:
@trbromley But it looks so much better with the curve!
More seriously, we've indeed decided to keep plot options minimal, in agreement with the philosophy we've followed so far with the `plot` module.